charts/k8ssandra/values.yaml

```diff
@@ -533,6 +566,20 @@ medusa:
   # -- Enable Medusa resources as part of this release. If enabled,
   # `bucketName` and `storageSecret` **must** be defined.
   enabled: false
+
+  # -- Security context override for Medusa container.
+  # K8ssandra defaults readOnlyRootFilesystem: true
+  # TODO - overriding to disable default for now, due to medusa write perm needed
+  securityContext:
```

Comment:
Are there security settings that won't work with this container? If so we should either document it or not allow it. I'm fine with just documenting it and doing that in a follow PR if that works for you.